lib/matplotlib/axes/_base.py

```diff
@@ -568,11 +568,17 @@ def __init__(self, fig, rect,
             right=rcParams['ytick.right'] and rcParams['ytick.major.right'],
             which='major')
 
+        self.layoutbox = None
+        self.poslayoutbox = None
```

Comment:
Should these have leading underscores?  Or are they intended to be part of the public API?

Reply:
Well, a) when I started, I didn't even know about `_privatestuff`, so this naming was in ignorance.  b) afterwards, I kind of thought that there may be occasion the knowledgable user would want access to these.  However, I think I agree that we don't want this to be public because it could change, so I'll set accordingly.